wordpress-mobile/WordPress-Android
# Self-hosted userlist

This PR adds a screen for the self-hosted user list. Note that dummy data is being used here. Actual data requested via `wordpress-rs` will come later.

To test:

* Enable `self_hosted_users` in debug settings
* Switch to a self-hosted site
* Note that a new "Users" item appears
* Tap on that and verify the screen with dummy data looks fine
* Switch to a non-self-hosted site and verify the "Users" item doesn't appear
* View the various Compose previews of the `UserListScreen` and verify they look fine (note that remote avatar images won't load in the preview)

This is ready for review but should not be merged because I use an image from my blog in the preview. I'll remove that after the PR is approved and then I'll merge it.

![users](https://github.com/user-attachments/assets/9edb6372-f811-4720-8a7e-4e8854350b7e)
